src/Routes/Books/Book.jsx

```diff
@@ -10,12 +12,22 @@ const Book = ({ idBook, book }) => {
   } = book;
   return (
     <>
-      <div className="card mb-3">
-        <div className="card-body">
-          <h6 className="card-title">{category}</h6>
-          <h4 className="card-text">{title}</h4>
-          <p className="text-muted">{author}</p>
-          <ActionsBook id={idBook} />
+      <div className="card mb-3 rounded-0">
+        <div className="card-body p-4">
+          <div className="row">
+            <div className="col-12 col-md-6">
+              <h6 className="card-title fw-bold">{category}</h6>
+              <h4 className="card-text fw-bold">{title}</h4>
+              <p className="text-muted fw-bold">{author}</p>
+              <ActionsBook id={idBook} />
+            </div>
+            <div className="col-4 col-md-3 mt-3">
+              <Progress />
+            </div>
+            <div className="col-8 col-md-3 mt-3">
+              <Chapter />
+            </div>
+          </div>
         </div>
       </div>
```

Comment:
- [ ] Kindly improve the styling of the book. We are to style the component as exactly as it is in the Zeplin design. Kindly compare the pictures below and try to improve your font styles.

- [ ] You are missing the demarcation (vertical straight line) between the left and right sections of the boo card as we have on the design.

- [ ] Kindly ensure the font on the progress bar section (text with % sign) should match the design font. The progress bar should also be moved further away from the right section to the left.

- [ ] Kindly match the font on the leftmost section to the design, including the text size. The section listing the links should be blue in color as we have on the design.

- [ ] The section with the update progress button should have matching font with the design.

- [ ] Kindly improve the navbar font as well. It should also be smaller.

  Your Implementation | Zeplin Design
  :---:|:---:
  ![ruwa](https://user-images.githubusercontent.com/54848363/222881853-9077c841-6db7-4d0c-b42e-93e66c817b4b.jpg) | ![dioq](https://user-images.githubusercontent.com/54848363/222881855-c6a3e8cd-a721-4a6e-8f25-0e9c9584d223.jpg)

Reply:
Thanks for the information, I am changing the style to match with template